umbraco/Umbraco-CMS
# V14: Resize url endpoint

# Notes
- Added `imaging/resize/urls` endpoint
- Added `ResizeImageFactory` to convert media into these resize URLs.
- Added logic to only return URLs for allowed files

# How to test
- Add some media (drag some image files into the media section, this should add media
- Call the new endpoint with the ids of the media
- Go to the URL the endpoint returns, it should be your resized image 🙌
